src/dmd/denum.d

```diff
@@ -374,6 +375,7 @@ extern (C++) final class EnumMember : VarDeclaration
     Type origType;
 
     EnumDeclaration ed;
+    bool isdeprecated;
```

Comment:
FYI: We now have an official [D style](https://dlang.org/dstyle.html) which we try to follow for new variables.
So this would be `isDeprecated`

Reply:
Base class Dsymbol already has a virtual method `isDeprecated()`. And some classes e.g. `AggregateDeclaration` has a field `isdeprecated`, which is returned by overridden method `isDeprecated()`.
Maybe stay it as is?

Reply:
Hmm, I guess `_deprecated` or `_isDeprecated` or `isDeprecatedEnum` would be better than having a symbol which only differences in the spelling.
BTW `isdeprecated` is only set once and never looked at, which looks a bit suspicious.

Reply:
Ok, I've just removed it:)